lib/sentry/event.ex

```diff
@@ -78,7 +78,6 @@ defmodule Sentry.Event do
           sdk: Interfaces.SDK.t() | nil,
           threads: [Interfaces.Thread.t()] | nil,
           user: Interfaces.user() | nil,
-
```

Comment:
Can you leave these lines in? Not really a necessary change here 🙃 Alos the one below.